### PR TITLE
Publish rib-android

### PR DESCRIPTION
**Pull request by lukaville**
**Description**:

**Related issue(s)**:

### CODE REVIEW

File: android/libraries/rib-android/gradle.properties [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#
+# Copyright (C) 2017. Uber Technologies
```
Comment (ShikaSD) [inline android/libraries/rib-android/gradle.properties]: Should we use copyright of Uber here?

File: android/libraries/rib-android/gradle.properties [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+POM_NAME=RIBs (Android)
+POM_ARTIFACT_ID=rib-android
+POM_PACKAGING=android
```
Comment (ShikaSD) [inline android/libraries/rib-android/gradle.properties]: Just curious, what is resulting package name?

Comment (lukaville) [reply android/libraries/rib-android/gradle.properties]: Group is `GROUP=com.badoo.ribs`
this option is not about package name, it's to specify that it's an android module :)

File: android/libraries/rib-android/gradle.properties (hunk 1 above)
Comment (lukaville) [reply android/libraries/rib-android/gradle.properties]: not sure looks like we should not remove it
image

Comment (ShikaSD) [reply android/libraries/rib-android/gradle.properties]: I would double check, but let's leave it for now :)

File: android/libraries/rib-android/gradle.properties (hunk 2 above)
Comment (ShikaSD) [reply android/libraries/rib-android/gradle.properties]: Yeah, I just did not notice the group name, so was confused a bit =)

**Pull request merged**